<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Iggy Zomboid</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Igg Serial Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="output_text">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>771</width>
      <height>391</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="connectionBox">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>621</width>
      <height>148</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>57</y>
       <width>163</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>View Mode:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Text</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">view_radio_buttons_group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Hex</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">view_radio_buttons_group</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>85</y>
       <width>135</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Line ending:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="lineEnding_comboBox"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>115</y>
       <width>123</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Baud rate:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="baud_comboBox"/>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="port_comboBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>481</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>520</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>25</y>
       <width>25</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh_ports_Button">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="connect_Button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="view_radio_buttons_group"/>
 </buttongroups>
</ui>
